data/json/overmap/overmap_special/municipal_reactor.json

```
@@ -0,0 +1,26 @@
+[
+  {
+    "id": "municipal_reactor",
+    "type": "overmap_special",
+    "connections": [ { "point": [ -1, 1, 0 ], "from": [ 0, 1, 0 ], "connection": "local_road" } ],
+    "overmaps": [
+      { "point": [ 0, 0, 0 ], "overmap": "municipal_reactor_north" },
+      { "point": [ 0, 0, -1 ], "overmap": "municipal_reactor_b_north" },
+      { "point": [ 0, 0, -2 ], "overmap": "municipal_reactor_b2_north" },
+      { "point": [ 0, 0, -3 ], "overmap": "municipal_reactor_b3_north" },
+      { "point": [ 1, 0, 0 ], "overmap": "municipal_reactor_ware_north" },
+      { "point": [ 1, 0, -1 ], "overmap": "municipal_reactor_ware_b_north" },
+      { "point": [ 0, 1, 0 ], "overmap": "road_ew" },
+      { "point": [ 1, 1, 0 ], "overmap": "road_end_east" },
+      { "point": [ 0, 2, 0 ], "overmap": "park_north" },
+      { "point": [ 1, 2, 0 ], "overmap": "office_cubical_north" },
+      { "point": [ -1, 2, 0 ], "overmap": "pwr_sub_s_north" },
+      { "point": [ -1, 0, 0 ], "overmap": "s_lot" }
+    ],
+    "locations": [ "wilderness", "swamp" ],
+    "city_distance": [ -1, 6 ],
+    "city_sizes": [ 2, 14 ],
+    "occurrences": [ 0, 4 ],
+    "flags": [ "ELECTRIC_GRID" ]
```

Vanilla power plants only have 0-2 occurrences and a less wacky range of allowed cities. Might be better as a `UNIQUE` special? Suggestion otherwise copies the location data of power plants.
```suggestion
    "locations": [ "land" ],
    "city_distance": [ 2, 25 ],
    "city_sizes": [ 3, -1 ],
    "occurrences": [ 50, 100 ],
    "flags": [ "CLASSIC", "URBAN", "ELECTRIC_GRID", "UNIQUE" ]
```

---

Setting it as unique is fine. Nuke plants are incredibly rare in Mainland USA due to uh, a lot of reasons and history. An example of "I had no idea what most of these fields did and thought Aftershocks values were okay or someone would point out how to improve it."